java/java.lsp.server/vscode/README.md

```diff
@@ -1,4 +1,4 @@
-# NBCode: A NetBeans based Extension for VSCode
+# VSNetBeans
```

Comment:
IMHO title should correspond to the description in package.json: Apache NetBeans Language Server Extension for VS Code

Reply:
OK, updated as you suggest in 1e18194570